analyses/summary/helpers.go

```diff
@@ -0,0 +1,53 @@
+package summary
```

Comment:
Can you add a test to verify the summary ?

Reply:
Ack. Added Unit Test